### ISSUE TITLE
Error when branch name == file name

### ISSUE THREAD

**Issue by letmaik** [mhutchie/vscode-git-graph#163]
**Describe the Bug**
```
Error: Unable to load commits
fatal: ambiguous argument 'foo': both revision and filename
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
```

**Steps to Reproduce**
Steps to reproduce the behaviour:
1. Create branch "foo"
2. Create file "foo" and add/commit
3. Select "foo" in Branches dropdown of Git Graph

**Expected Behaviour**
No error.

**Environment**
 - Git Graph Extension Version: 1.13.0
 - Visual Studio Code Version: 1.37.1
 - Operating System: Windows 10

**Comment by mhutchie** [comment]
Thanks for identifying this! I’ll have a fix for this available soon.

**Comment by mhutchie** [comment]
This has been fixed, and will be available in v1.14.0.

If you'd like to use it before the next release, you can download [v1.14.0-beta.0](https://github.com/mhutchie/vscode-git-graph/releases/tag/v1.14.0-beta.0), and install it following the instructions provided [here](https://github.com/mhutchie/vscode-git-graph/wiki/Downloading-&-Installing-Releases#manual-installation).
